drivers/media/i2c/imx290.c

```diff
@@ -1091,6 +1106,12 @@ static s64 imx290_check_link_freqs(const struct imx290 *imx290,
 	return 0;
 }
 
+static const struct of_device_id imx290_of_match[] = {
+	{ .compatible = "sony,imx290", .data = (void *)0 },
+	{ .compatible = "sony,imx290-mono", .data = (void *)1 },
+	{ /* sentinel */ }
```

Comment:
Oh look - a initialising a structure to zero without the aid of a zero.

Reply:
:-P